aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/Startup.cs

```diff
@@ -32,100 +32,99 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<AppDbContext>(options => 
                 options.UseInMemoryDatabase("InMemoryDb"));
 
-            services.AddRazorPages()
-                .AddRazorPagesOptions(options =>
-                {
-                    #region snippet1
-                    options.Conventions.Add(new GlobalTemplatePageRouteModelConvention());
-                    #endregion
+            services.AddRazorPages(options =>
+            {
+                #region snippet1
+                options.Conventions.Add(new GlobalTemplatePageRouteModelConvention());
+                #endregion
 
-                    #region snippet2
-                    options.Conventions.Add(new GlobalHeaderPageApplicationModelConvention());
-                    #endregion
+                #region snippet2
+                options.Conventions.Add(new GlobalHeaderPageApplicationModelConvention());
+                #endregion
 
-                    #region snippet3
-                    options.Conventions.AddFolderRouteModelConvention("/OtherPages", model =>
+                #region snippet3
+                options.Conventions.AddFolderRouteModelConvention("/OtherPages", model =>
+                {
+                    var selectorCount = model.Selectors.Count;
+                    for (var i = 0; i < selectorCount; i++)
                     {
-                        var selectorCount = model.Selectors.Count;
-                        for (var i = 0; i < selectorCount; i++)
+                        var selector = model.Selectors[i];
+                        model.Selectors.Add(new SelectorModel
                         {
-                            var selector = model.Selectors[i];
-                            model.Selectors.Add(new SelectorModel
+                            AttributeRouteModel = new AttributeRouteModel
                             {
-                                AttributeRouteModel = new AttributeRouteModel
-                                {
-                                    Order = 2,
-                                    Template = AttributeRouteModel.CombineTemplates(
-                                        selector.AttributeRouteModel.Template, 
-                                        "{otherPagesTemplate?}"),
-                                }
-                            });
-                        }
-                    });
-                    #endregion
+                                Order = 2,
+                                Template = AttributeRouteModel.CombineTemplates(
+                                    selector.AttributeRouteModel.Template, 
+                                    "{otherPagesTemplate?}"),
+                            }
+                        });
+                    }
+                });
+                #endregion
```

Comment:
I haven't changed any code here; just dedented, It seems to have confused the diffing.